app/nfc.c

```
@@ -358,90 +160,89 @@ int nfc_init(void)
             else
             {
                 nfc_data_out_len = 2;
-                memcpy(nfc_data_out_buf,"\x6D\x00",nfc_data_out_len); 
+                memcpy(nfc_data_out_buf, "\x6D\x00", nfc_data_out_len);
             }
         }
     }
 }
 
-void nfc_poll(void *p_event_data,uint16_t event_size)
+void nfc_poll(void* p_event_data, uint16_t event_size)
```

Cppcheck (2.10-2)

[style] The function 'nfc_poll' is never used.

(unusedFunction)